23_12_week2/donghyun/20291.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+using namespace std;
+
+int main () {
+	ios_base::sync_with_stdio(0);cin.tie(0);cout.tie(0);
+
+	int n;
+	cin >> n;
+	vector<string> input(n);
+	map<string, int> m;
+	for (int i = 0;i < n; i++)
+		cin >> input[i];
+
+	for (int i = 0; i < n; i++) {
+		string s = input[i].substr(input[i].find('.') + 1);
+		if (m.find(s) != m.end())
+			m[s] += 1;
+		else
+			m[s] = 1;
```

Comment:
```suggestion
		m[s]++;
```

그냥 이렇게 해도 된다는 사실 알고 계신가용 ?